package/MDAnalysis/analysis/align.py

```diff
@@ -745,9 +745,8 @@ def rms_fit_trj(
         # so that R acts **to the left** and can be broadcasted; we're saving
         # one transpose. [orbeckst])
         rmsd[k] = qcp.CalcRMSDRotationalMatrix(
-            ref_coordinates.T.astype(
-                np.float64), traj_coordinates.T.astype(
-                np.float64), natoms, rot, weights)
+            ref_coordinates.astype(np.float64), 
```

Comment:
@rbrtdlgd tried it and once you see the benchmarks, you'll see that it makes little difference. On the other hand, the precision of the final results can drop down to 1e-4 or maybe even 1e-3 with float32 (as opposed to 1e-5 to 1e-7 or better with float64). There might be a way to identify the exact part in the qcprot algorithm where float64 is needed but the difference in speed that we saw did not make this worthwhile. So internally we decided to only change the transpose because that had a sizable impact.